interfaces/builtin/basedeclaration_test.go

```diff
@@ -224,6 +224,78 @@ func (s *baseDeclSuite) TestAutoConnectionContent(c *C) {
 	cand := s.connectCand(c, "content", "", "")
 	err := cand.CheckAutoConnect()
 	c.Check(err, NotNil)
+
+	slotDecl1 := s.mockSnapDecl(c, "slot-snap", "slot-snap-id", "pub1", "")
+	plugDecl1 := s.mockSnapDecl(c, "plug-snap", "plug-snap-id", "pub1", "")
+	plugDecl2 := s.mockSnapDecl(c, "plug-snap", "plug-snap-id", "pub2", "")
+
+	// same publisher, same content
+	cand = s.connectCand(c, "stuff", `name: slot-snap
+slots:
+  stuff:
+    interface: content
+    content: mk1`, `name: plug-snap
```

Comment:
Agree with jdstrand.. this is super hard to read properly. Can we please make it:
```
cand = s.connectCand(c, "stuff", `
name: slot-snap
...
`, `
name: plug-snap
...
`)